### PR TITLE
Add cost management api client

### PR DESCRIPTION
**Pull request by rvsia**
**Description**

This is a client library for [RedHatInsights/Cost Management](https://github.com/project-koku).

@Hyperkid123 
@karelhala

### CODE REVIEW

File: packages/cost-management/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+parserOptions:
```
Comment (Hyperkid123) [inline packages/cost-management/.eslintrc.yml]: Eslint is not needed i think. Code is purely generated.

File: packages/cost-management/.npmignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+typings
+
+test
+*.sh
+
+jest.config.js
+.openapi-generator
+tsconfig.json
+.eslintrc.yml
+.openapi-generator-ignore
```
Comment (Hyperkid123) [inline packages/cost-management/.npmignore]: i think that folder `.openapi-generator` is missing.

**Comment by rvsia** [comment]
Updated the openapi link

**Pull request merged**